src/signals/IncidentMap/components/IncidentMap/IncidentMap.tsx

```diff
@@ -49,22 +52,23 @@ export const IncidentMap = () => {
     },
     [setMapMessage, setShowMessage]
   )
+
   /* istanbul ignore next */
-  const handleIncidentSelect = useCallback((incident) => {
+  const handleIncidentSelect = useCallback((incident: Incident) => {
     setSelectedIncident(incident)
     setDrawerState(DrawerState.Open)
   }, [])
```

Comment:
no deps?

Reply:
it doesnt have any since incident is a prop.